### DISCUSSION TITLE
Using Makefiles or Just

### DISCUSSION THREAD

**Discussion by janaakhterov** [hashgraph/hedera-sdk-java#956] [Ideas]
Current all the SDKs have their own build system because they're in different languages. However, while writing the core I was using Makefiles, and I really liked that although each SDK was different you can easily build, test, or clean using a single command. Would it make sense to enforce this now in each SDK? This also _sort of_ cleans up your terminal history if you use a terminal. 
cc @rbair23 @SimiHunjan @Sean-Tedrow-LB @andrix10 @keirabee @easpaas @rickyrodmac

**Comment by rickyrodmac** [comment]
I have not used a makefile since the early 2000's. They work well and I really like the side-affect of keeping a cleaner history.
